### ISSUE TITLE
readyNow method when using horizon with this package

### ISSUE THREAD

**Issue by bretto36** [tenancy/tenancy#296]
So it still appears like #268 is still an issue even in laravel 10. 

In #268 the final comments suggested a solution. 

If i load the horizon/api/workload endpoint which is loaded via axios on the horizon page it fails with the missing readyNow method

I've successfully tested it locally by replacing the ->booted call with callAfterResolving which then loads stuff in correct order

$this->callAfterResolving(QueueManager::class, function (QueueManager $queue) {

In the spatie/laravel-ignition package there is a line here 

$queue = $this->app->get('queue');

When this is called it actually calls the tenancy stuff BEFORE the horizon stuff, where as on every other web request it calls the horizon code first then the tenancy code (HorizonServiceProvider then Tenancy\Identification\Drivers\Queue\Providers\IdentificationProvider)

I have confirmed the above changes DOES work and the queue is processed as normal. I will submit a PR

**Comment by ArlonAntonius** [comment]
@bretto36 Is this purely because of the Combination Tenancy, Laravel Horizon and Spatie Laravel Ignition or does this also happen without those packages?

**Comment by bretto36** [comment]
@ArlonAntonius yes the packages don't seem to play nice. They both register things in a different way

**Comment by bretto36** [comment]
@ArlonAntonius ignition is a default package in the laravel/laravel composer file. Just in devDependencies not in regular dependencies

**Comment by ArlonAntonius** [comment]
@bretto36 Do you by chance have an example repo? Will look into this myself as well, but in case I bump into a deadend, it would be nice to know that there is an example where this is clearly broken
